misc/host_tests.conf

```diff
@@ -12,7 +12,6 @@ add mudgee
 
 # Additional base modules
 include subset/pentests/build.conf
-include subset/network/build.conf
```

Comment:
this is being removed, but still used below?  What's the plan here?

Reply:
The plan is that I fix net macoui, network and bacnet tests after this nmap test is merged to master.